docs/source/installation.rst

```diff
@@ -21,49 +21,159 @@ Requirements
 * `Requests <https://requests.readthedocs.io/>`_: used to get data from websites
 * `Pecos v0.1.9 <https://pecos.readthedocs.io/>`_: used for quality control analysis
 
-
 Install MHKiT-Python
 ^^^^^^^^^^^^^^^^^^^^^
 
-Option 1: PIP Install from Python
-""""""""""""""""""""""""""""""""""""
+Option 1: Using Conda
+"""""""""""""""""""""
 
-This option is recommended for MHKiT-Python users.
-To install MHKiT-Python using `pip <https://pip.pypa.io/en/stable/>`_::
+For most users, `Conda <https://docs.conda.io/projects/conda/en/stable/index.html>`_ is the preferred choice for installing MHKiT-Python. Conda includes the ``conda`` command line utility to manage Python versions and dependencies within `environments <https://business-docs.anaconda.com/en/latest/user/environment.html#:~:text=An%20environment%20is%20a%20folder,from%20other%20collections%20of%20packages.>`_. Anaconda provides a standardized Python build environment that is compatible across operating systems, simplifying MHKiT-Python installation and setup for all users.
 
-	pip install mhkit
 
-Using this option to install MHKiT-Python will automatically install Pecos.
+1. `Install Conda <https://docs.conda.io/projects/conda/en/stable/user-guide/install/index.html>`_ with Anaconda Distribution or Miniconda:
 
+   - `Anaconda Distribution <https://docs.anaconda.com/anaconda/install/>`_ is a full featured installer that comes with a suite of packages for data science, as well as Anaconda Navigator, a GUI application for working with conda environments.
+   - `Miniconda <https://docs.anaconda.com/miniconda/#quick-command-line-install>`_ is a minimal installer provided by Anaconda that installs the ``conda`` command line utility and its dependencies.
 
-Option 2: Clone Repository from GitHub
-""""""""""""""""""""""""""""""""""""""""""
 
-This option is recommended for MHKiT-Python developers.
-To install MHKiT-Python using `git <https://git-scm.com/>`_::
 
-	git clone https://github.com/MHKiT-Software/MHKiT-Python
-	cd mhkit-python
-	pip install -e .
+2. Access the Conda ``base`` environment:
+
+Using Anaconda Distribution:
+
+- Launch the "Anaconda Navigator" application
+
+- Navigate to the "Environments" section
+
+.. image:: ./figures/install_anaconda_select_environment_section.png
+  :width: 500
+  :alt: Navigating to the Anaconda Navigator "Environments" section
+
+- Click the play button next to ``base(root)`` and select "Open Terminal"
+
+.. image:: ./figures/install_anaconda_open_base_environment_terminal.png
+  :width: 500
+  :alt: Opening the terminal for the ``mhkit`` environment
+
+
+Using Miniconda:
+
+- Launch your preferred terminal and execute the following command:
```

Comment:
I don't think conda is guaranteed to be active in any terminal (e.g. command prompt). I think we direct users to search for and open the anaconda terminal directly